packages/atomic/src/components/search/atomic-text/atomic-text.ts

```diff
@@ -23,6 +36,17 @@ export class AtomicText extends LitElement {
       }),
   };
 
+  static styles: CSSResultGroup = [
+    TailwindLitElement.styles,
+    css`
+      div {
+        border: 1px solid red;
+        border-radius: var(--atomic-border-radius-xl);
+      }
+    `,
+    unsafeCSS(styles),
+  ];
+
```

Comment:
can't use @apply rules in inline stylesheets
```css
@apply rounded-xl;
```
Can still use css variables, but I don't see a good reason to do this instead of an external stylesheet or tailwind classes...

Reply:
If a component does not have a separate stylesheet, no need to add this, as long as the component extends TailwindLitElement tailwind classes will work!